### PR TITLE
Update dockerfile-jvm.ftl

### PR DESCRIPTION
**Pull request by survivant**
fix the issue : JAVA_DEBUG fix for Java 11

### CODE REVIEW

File: independent-projects/tools/common/src/test/resources/templates/dockerfile-jvm.ftl [hunk 1]
```diff
@@ -40,4 +40,4 @@ COPY ${build_dir}/*-runner.jar /deployments/app.jar
 EXPOSE 8080
 USER 1001
 
-ENTRYPOINT [ "/deployments/run-java.sh" ]
\ No newline at end of file
+ENTRYPOINT [ "/deployments/run-java.sh" ]
```
Comment (geoand) [inline independent-projects/tools/common/src/test/resources/templates/dockerfile-jvm.ftl:43]: This line change doesn't seem to change anything, can you revert it please?

Comment (survivant) [reply independent-projects/tools/common/src/test/resources/templates/dockerfile-jvm.ftl:43]: is it better now ?

Comment (geoand) [reply independent-projects/tools/common/src/test/resources/templates/dockerfile-jvm.ftl:43]: The change still shows up in the GitHub UI.

Moreover, once the proper revert is in place, we'll need the commits to be squashed

Comment (survivant) [reply independent-projects/tools/common/src/test/resources/templates/dockerfile-jvm.ftl:43]: I did the modifs from github UI.  I'll just do it from Intellij.

**Comment by survivant** [comment]
here the new PR  https://github.com/quarkusio/quarkus/pull/9376

**Pull request closed without merging**